src/otx/core/model/module/base.py

```diff
@@ -159,10 +167,32 @@ def ensure_list(item: Any) -> list:  # noqa: ANN401
                 lr_scheduler_config["interval"] = scheduler.interval
             if hasattr(scheduler, "monitor"):
                 lr_scheduler_config["monitor"] = scheduler.monitor
-            lr_schedulers.append(lr_scheduler_config)
 
+            lr_schedulers.append(lr_scheduler_config)
         return optimizers, lr_schedulers
 
+    def optimizer_step(
+        self,
+        epoch: int,
+        batch: int,
+        optimizer: torch.optim.Optimizer,
+        closure: Callable[[], Tensor | None],
```

Comment:
```suggestion
        optimizer_closure: Callable[[], Tensor | None],
```

https://github.com/Lightning-AI/pytorch-lightning/blob/f23b3b1e7fdab1d325f79f69a28706d33144f27e/src/lightning/pytorch/core/module.py#L1270

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/3035/commits/fa86c3bd4f47fa8e8d5de6f42d3f5d662415db83